canonical-web-and-design/ubuntu.com
# Appliance hardware

## Done
Build the appliance hardware page

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/hardware
- Check the copy matches [the doc](https://docs.google.com/document/d/179RBNgtGr5Y8CkkYaUa6sdFaaTJTTvs_VCTr6UzECFo/edit#)
- Check it matches [the design](https://app.zeplin.io/project/5eb3d16a91c4117789940141/screen/5ee2140b4b123fb826c6ef8d)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/7710

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/84504550-079c5c80-acb4-11ea-867e-83347e4c63f3.png)
